scripts/setup_db.py

```diff
@@ -33,45 +37,39 @@ def create_tables(hostname, dbname, user):
             )
             """,
             """
-            DROP TABLE IF EXISTS service_providers
-            """,
-            """
             CREATE TABLE service_providers (
-                id INTEGER PRIMARY KEY,
-                census_division_id INTEGER REFERENCES census_division (id),
-                name TEXT NOT NULL,
-                website TEXT,
-                report_year INTEGER,
-                report_link TEXT,
-                expenses INTEGER,
+                id           INTEGER PRIMARY KEY,
+                name         TEXT NOT NULL,
+                website      TEXT,
+                report_year  INTEGER,
+                report_link  TEXT,
+                expenses     INTEGER,
                 client_total INTEGER,
-                staff_total INTEGER,
-                address TEXT,
+                staff_total  INTEGER,
+                notes        TEXT
+            )
+            """,
+            """
+            CREATE TABLE sp_census_divisions (
+                id                 INTEGER PRIMARY KEY,
+                sp_id              INTEGER REFERENCES service_providers (id),
+                census_division_id INTEGER REFERENCES census_division (id)
+            )
+            """,
+            """
+            CREATE TABLE sp_locations (
+                id        INTEGER PRIMARY KEY,
+                sp_id     INTEGER REFERENCES service_providers (id),
+                address   TEXT,
                 longitude DECIMAL(16,14),
-                latitude DECIMAL(16,14),
-                notes TEXT
+                latitude  DECIMAL(16,14),
+                main      BOOLEAN
```

Comment:
what is this column used for? the name is a bit ambiguous

Reply:
It's a bool to indicate if the location is the service provider's main location.